src/libs/tmdk/e2e-tests/basic-e2e/basic-e2e-iam.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export const twinMakerPermissionPolicySuffix = "PermissionPolicy";
+
+export const twinMakerAssumeRolePolicy = {
+  Version: "2012-10-17",
+  Statement: [
+    {
+      Effect: "Allow",
+      Principal: {
+        Service: "iottwinmaker.amazonaws.com",
+      },
+      Action: "sts:AssumeRole",
+    },
+  ],
+};
+
+export const twinMakerPermissionPolicy = {
+  Version: "2012-10-17",
+  Statement: [
+    {
+      Effect: "Allow",
+      Action: [
+        "s3:GetBucket",
+        "s3:GetObject",
+        "s3:ListBucket",
+        "s3:PutObject",
+        "s3:ListObjects",
+        "s3:ListObjectsV2",
+        "s3:GetBucketLocation",
+      ],
+      Resource: [
+        "s3ArnStar", // replaced with workspace bucket
```

Comment:
maybe make these more distinctive as something to be replaced? (e.g. `___PLACEHOLDER_FOR_WORKSPACE_BUCKET___`)

Reply:
Will do